ansible/roles/create-vm/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+# create-vm tasks
+
+- name: list all VMs
+  community.libvirt.virt:
+    command: list_vms
+  register: all_vms
+
+- name: find VMs that start with "microshift-*"
+  set_fact:
+    vm_match: "{{ all_vms.list_vms | join(' ') | regex_findall('microshift-\\S+', multiline=True, ignorecase=True) }}"
+
+- name: destroy microshift VM
```

Comment:
I'm a little uncomfortable with this destroying VMs it discovers, rather than being told explicitly which VMs to manage. Could the variable defining the VM settings be a list the user controls? That would, for example, let  me create a RHEL 8 VM with 4 CPUs and a RHEL 9 VM with 2 CPUs.

If you like the idea, we can leave it for future enhancement.